timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -435,11 +438,9 @@ export default {
       this.$store.dispatch('updateContextLinks')
       this.$store.dispatch('updateAnalyzerList', this.sketchId)
       this.loadingSketch = false
-      this.showLeftPanel = true
-      this.$nextTick(function () {
-        this.setDrawerBorderStyle()
-        this.setDrawerResizeEvents()
-      })
+      if (this.hasTimelines) {
```

Comment:
```suggestion
      if (this.hasTimelines && !this.isArchived) {
```

To close the left panel on archived sketches per default. (Still allows them to be opened on an archived sketch though)